src/main/java/com/devathon/preguntonic/model/Game.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Game {
+
+  private UUID id;
+  private Room room;
```

Comment:
I think it is no needed here, and we need  another field: `list<Question> questions`

Reply:
Ups! I forgotten that... Sorry. A question of that... how we will manage the entity references in our local storage (we have a M-N table that links an entity from de database and an object from ephemeral storage)